src/containers/AccountConnection.jsx

```diff
@@ -251,73 +252,42 @@ class AccountConnection extends Component {
-                  }
-                </DescriptionContent>
-            }
-            {success
-              ? <div>
-                <DescriptionContent
-                  title={t(`account.success.title.${success.type}`, { name: connector.name })}
-                  messages={success.messages}
-                >
-                  {Array.isArray(connector.dataType) && connector.dataType.includes('bill') &&
-                    <p>
-                      {t(`account.message.${success.type === SUCCESS_TYPES.TIMEOUT ? 'syncing' : 'synced'}.bill`, { name: connector.name })}
-                      <br />
-                      <span className={styles['col-account-success-highlighted-data']}>
-                        {this.state.account.auth.folderPath}
-                      </span>
-                    </p>
-                  }
-                </DescriptionContent>
-                <AccountLoginForm
-                  onAccountConfig={() => this.goToConfig()}
-                  onCancel={() => this.cancel()}
-                  isSuccess={!!success}
-                />
-              </div>
-              : <AccountLoginForm
-                connectorSlug={connector.slug}
-                isOAuth={connector.oauth}
-                oAuthTerminated={oAuthTerminated}
-                fields={fields}
-                submitting={submitting}
-                disableSuccessTimeout={this.props.disableSuccessTimeout}
-                deleting={deleting}
-                values={editing && account ? account.auth || account.oauth : {}}
-                error={error && error.message === ACCOUNT_ERRORS.LOGIN_FAILED}
-                forceEnabled={!!error}
-                onDelete={() => this.deleteAccount()}
-                onSubmit={(values) => this.submit(Object.assign(values, {folderPath: t('account.config.default_folder', connector)}))}
-                onCancel={() => this.cancel()}
-              />
-            }
+            { hasGlobalError && <DescriptionContent
+              cssClassesObject={{'coz-error': true}}
+              title={t('account.message.error.global.title')}
+              messages={[t('account.message.error.global.description', {name: connector.name})]}
+            /> }
+
+            <KonnectorSync
+              date={account && account.lastSync}
+            />
+
+            <KonnectorAccount
+              connector={connector}
+              account={account}
+              fields={fields}
+              success={success}
+              editing={editing}
+              disableSuccessTimeout={this.props.disableSuccessTimeout}
+              oAuthTerminated={oAuthTerminated}
+              submitting={submitting}
+              deleting={deleting}
+              error={error}
+              onDelete={() => this.deleteAccount()}
+              onSubmit={(values) => this.submit(Object.assign(values, {folderPath: t('account.config.default_folder', connector)}))}
+              onCancel={() => this.cancel()}
+            />
```

Comment:
Oh you're right, sorry for that, I am looking why I don't have the description then.